app/src/forms/email/emailService.js

```diff
@@ -182,15 +188,19 @@ const service = {
       };
       return service._sendEmailTemplate('sendSubmissionReceived', configData, submission, referer);
     } catch (e) {
-      log.error(`formId: ${formId}, submissionId: ${submissionId}, body: ${JSON.stringify(body)}, referer: ${referer}`, { function: 'submissionConfirmation' });
-      log.error(e.message, { function: 'submissionReceived' });
-      log.error(e);
+      log.error(e.message, e, {
+        function: 'submissionReceived',
+        formId: formId,
+        submissionId: submissionId,
+        body: JSON.stringify(body),
```

Comment:
It may be fine to leave the body as a JSON object directly, as stringifying it will likely add a bunch of character encoding wrappers on the content. Winston can handle direct object references while logging in JSON mode.

Reply:
removing the JSON.stringify call on the body